conf/solr/8.8.1/updateSchemaMDB.sh

```diff
@@ -56,27 +56,67 @@ if [ ! -z "${UNBLOCK_KEY}" ]; then
   fi
 fi
 
+# Check if schema.xml file exists
+if [ ! -f "${TARGET}/schema.xml" ]; then
+  echo "'schema.xml' file not found in target dir '${TARGET}'"
+  usage
+  exit 0
+fi
+
 ### Retrieval
 echo "Retrieve schema data from ${DATAVERSE_URL}/api/admin/index/solr/schema"
 TMPFILE=`mktemp`
 curl -f -sS "${DATAVERSE_URL}/api/admin/index/solr/schema${UNBLOCK_KEY}" > $TMPFILE
 
 ### Fail gracefull if Dataverse is not ready yet.
-if [[ "`wc -l ${TMPFILE}`" < "3" ]]; then
+if [[ `wc -l < ${TMPFILE}` -lt 3 ]]; then
```

Comment:
We should start using `$()` here and let [`shellcheck`](https://www.shellcheck.net/) bark at other things it finds.